MODULE.bazel

```diff
@@ -295,25 +277,25 @@ http_archive(
     url = CLION_251_URL,
 )
 
+CLION_252_SHA = "ccf8a74534102d0985801e4ec1a5552ea283741ef8a7edefaced20dcf644612c"
+
+CLION_252_URL = "https://www.jetbrains.com/intellij-repository/snapshots/com/jetbrains/intellij/clion/clion/252.16512-EAP-CANDIDATE-SNAPSHOT/clion-252.16512-EAP-CANDIDATE-SNAPSHOT.zip"
```

Comment:
I just checked and found that we should not use candidates here as they can be overwritten too. here should be build with xx.yy.zz

Reply:
The issue is that both `252.16512.17-EAP-SNAPSHOT` and `252.13776.59-EAP-SNAPSHOT` are returning 404 for clion... For IJ they work correctly

Reply:
different products can't have the same build number